distributions/otelcol-otlp/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM alpine:3.19 as certs
+RUN apk --update add ca-certificates
+
+FROM scratch
+
+ARG USER_UID=10001
+USER ${USER_UID}
+
+COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
+COPY --chmod=755 otelcol-otlp /otelcol-otlp
+ENTRYPOINT ["/otelcol-otlp"]
+CMD ["--config", "/etc/otelcol-otlp/config.yaml"]
+EXPOSE 4317 55678 55679
```

Comment:
Do `55678 55679` need exposed? I would think only `4317 4318` is needed.

Reply:
You're right, fixed.